python/tank/util/process.py

```diff
@@ -62,6 +63,7 @@ def subprocess_check_output(*popenargs, **kwargs):
     if is_windows():
         process.stdin.close()
     output, unused_err = process.communicate()
+    output = six.ensure_str(output)
```

Comment:
As stated above, it seems saner to expect a string instead of a binary string out of the console. Thoughts?